workflow-init/profile/hawk.yml

```diff
@@ -21,6 +21,18 @@ parameters: &defaultParameters
   kustomize: bitbucket-fake
 
 component:
+  # This is an example single-module template, that you need to copy and modify according to specific project
```

Comment:
@hawk-bernhard-altendorfer Let's put the link to this into the README of the template so we will know what to update.